yarn.lock

```diff
@@ -3110,6 +3110,13 @@ bl@^1.0.0:
     readable-stream "^2.3.5"
     safe-buffer "^5.1.1"
 
+blockprotocol@0.0.1, blockprotocol@^0.0.1:
+  version "0.0.1"
+  resolved "https://registry.yarnpkg.com/blockprotocol/-/blockprotocol-0.0.1.tgz#2723d6b80f50c3da7cf4984b4e5824fc26570f76"
+  integrity sha512-neITCxiFVxM5S0C0KTPucMCB2Xip5xIYo3LZFXUY/vamUKM5Ov2viv39+og1X6Vfd/IZwuwONK3gjO6dtGDXDA==
+  dependencies:
+    "@types/react" "^17.0.38"
+
```

Comment:
I guess we need to update dependency in `site/package.json`. We should not depend on an external BP version.

Reply:
Done